harness/Adapter/index.tsx

```diff
@@ -213,18 +212,22 @@ class Adapter extends Component<IAdapterProps, IAdapterState> {
                 title="child-app"
               />
             </Grid>
-
             <Divider orientation="vertical" />
             <Grid item xs className={classes.fullHeight}>
               <Paper className={classes.fullHeight}>
-              
                 <Grid container justify="center">
                   <Grid item xs="auto">
                     <Box m={4}>
+                      <TextField select noAutoWidth onChange={this.handleInputChange}defaultValue="none" disableClearable disableSearchable>
```

Comment:
Seems like a there is a missing space before `defaultValue`.

Reply:
Are `disableClearable` and `disableSearchable` valid `TextField` props? I am thinking they might be left over props from the previous `Select`.

Reply:
good catch on the missing space and I thought about those two props as soon as I opened the PR as I was heading out for the evening. Updating now.